lib/utils/messages.js

```diff
@@ -206,16 +206,30 @@ module.exports = function () {
             prefix: ' ',
             message: 'was deleted within Anypoint Platform'
           }
-        }
+        },
+        rootRamlDeletedAction()
       ];
 
       var output = 'Conflicts found, since your last pull these changes were ' +
         'found:\n\n';
       output += buildStatusResponse(conflicts, actions);
-      output += '\n\nTo resolve run:\n';
-      output += '- ' + TOOL_NAME + ' pull (overwrites local changes)\n';
-      output += '- ' + TOOL_NAME + ' push -f (deletes remote API definition ' +
-        'files and replaces them with the local ones)';
+      output += '\n\n';
+      output += howToSolveConflicts(conflicts);
+
+      return output;
+    },
+    deletedRootRamlConflict: function (rootRamlPath) {
+      var output = 'The following conflict can\'t be solved with push -f\n\n';
+      var actions = [
+        rootRamlDeletedAction()
+      ];
+      var conflicts = {
+        rootRamlDeleted: rootRamlPath
```

Comment:
you could define this property as an array to avoid the typeof in buildStatusRespose